src/main/java/duke/commands/ListCommand.java

```diff
@@ -0,0 +1,58 @@
+package duke.commands;
+
+import java.util.ArrayList;
+
+import duke.data.TaskList;
+import duke.storage.Storage;
+import duke.tasks.Task;
+import duke.ui.Ui;
+
+/**
+ * This class represents a command to list out the tasks in the task list
+ * If a date is provided, tasks that fall on that date will be listed
+ */
+public class ListCommand extends Command {
+
+    private String date;
+
+    /**
+     * Constructs a new list command
+     *
+     * @param info Essential information for the output list
+     */
+    public ListCommand(String... info) {
+        super();
```

Comment:
A small thing but I don't think you have to call the superclass's constructor explicitly if it does not take in any arguments because by default it will always call `super()` if there is no call to the superclass's constructor :)